rest/functionsapitest/graphql_test.go

```diff
@@ -484,3 +484,79 @@ func TestInvalidGraphQLConfigurationValues(t *testing.T) {
 		assert.NoError(t, err)
 	})
 }
+
+// Test for GraphQL mutations via Guest user
+func TestGraphQLMutationsGuest(t *testing.T) {
+	rt := rest.NewRestTester(t, &rest.RestTesterConfig{GuestEnabled: true, EnableUserQueries: true})
+	if rt == nil {
+		return
+	}
+	defer rt.Close()
+	rt.DatabaseConfig = &rest.DatabaseConfig{DbConfig: rest.DbConfig{
+		GraphQL:       &kTestGraphQLConfig,
+		UserFunctions: &kTestGraphQLUserFunctionsConfig,
+	},
```

Comment:
is this some go fmt fault ? 
},
}

Reply:
Yep, go fmt automatically formats it this way. Can't really do anything about it.